helm-chart/values.yaml

```diff
@@ -92,3 +96,21 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+volumes: []
+# volumes:
+# - name: client-certificates
+#   secret:
+#     secretName: client-cert
+#     items:
+#     - key: ca.pem
+#       path: ca.pem
+#     - key: server.key
+#       path: server.key
+#     - key: server.pem
+#       path: server.pem
+
+volumeMounts: []
+# volumeMounts:
```

Comment:
same as above

Reply:
it may help someone who consumes the helm to see what the options are there without getting into the chart specs